### PR TITLE
Add new Azure functions example changes

### PR DESCRIPTION
**Pull request by xlight05**
## Purpose
$Subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by tharindu-nw — APPROVED** [review]

File: examples/azure-functions-deployment/az_deploy.out [hunk 1]
```diff
@@ -1,26 +1,12 @@
...
-"progress": "",
-"provisioningState": null,
-"received_time": "2020-07-02T06:47:56.2756472Z",
-"site_name": "functions1777",
-"start_time": "2020-07-02T06:47:56.7600364Z",
-"status": 4,
-"status_text": "",
-"url": "https://functions1777.scm.azurewebsites.net/api/deployments/latest"
-}
+$ func azure functionapp publish bal-bbe --script-root target/azure_functions
```
Comment (praneesha) [inline examples/azure-functions-deployment/az_deploy.out:1]:
```suggestion
$ func azure functionapp publish bal-bbe --script-root target/azure_functions
```
These commands should go to the `.md` file. Only the CLI output should be there in the `.out` file.

File: examples/azure-functions-deployment/azure_functions_deployment.bal [hunk 2]
```diff
@@ -1,28 +1,19 @@
...
+// This function gets triggered by an HTTP call with the name query parameter and returns a processed HTTP output to the caller.
+service / on new af:HttpListener() {
+    resource function get hello(string name) returns string {
+        return "Hello, " + name + "!";
     }
-    return "Blob: " + req.query["name"].toString() + " Length: " + 
-            length.toString() + " Content: " + blobIn.toString();
 }
 
 // This function gets executed every 10 seconds by the Azure function app. Once the function is executed, the timer
```
Comment (praneesha) [inline examples/azure-functions-deployment/azure_functions_deployment.bal:10]:
```suggestion
// This function gets executed every 10 seconds by the Azure Functions app. Once the function is executed, the timer 
```

File: examples/azure-functions-deployment/azure_functions_deployment.md [hunk 3]
```diff
@@ -12,10 +12,10 @@ Create a Ballerina package and replace the content of the generated BAL file wit
 Build the Ballerina program to generate the Azure Functions artifacts.
 ::: out bal_build.out :::
 
-Execute the Azure CLI command given by the compiler to publish the functions (replace with your respective Azure `<resource_group>` and `<function_app_name>`).
+Execute the Azure CLI command given by the compiler to publish the functions (replace with your respective Azure `<function_app_name>`).
```
Comment (praneesha) [inline examples/azure-functions-deployment/azure_functions_deployment.md:15]:
```suggestion
Execute the Azure CLI command given by the compiler to publish the functions (replace the sample app name given in the command with your respective Azure `<function_app_name>`).
```

File: examples/azure-functions-deployment/azure_functions_deployment.md [hunk 4]
```diff
@@ -12,10 +12,10 @@ Create a Ballerina package and replace the content of the generated BAL file wit
...
 ::: out az_deploy.out :::
 
 Invoke the `HTTP Trigger` functions.
 ::: out execute_function.out :::
 
-The `queuePopulationTimer` function is being triggered by the Azure Function App from a timer. You can check the  queue storage to see the output. For more information on the infrastructure, see [Azure Functions deployment](/learn/deployment/azure-functions/).
+The `timer` function is being triggered by the Azure Function App from a timer. You can check the queue storage to see the output. For more information on the infrastructure, see [Azure Functions deployment](/learn/deployment/azure-functions/).
```
Comment (praneesha) [inline examples/azure-functions-deployment/azure_functions_deployment.md:21]:
```suggestion
The `timer` function is triggered by the Azure Functions app from a timer. You can check the queue storage to see the output. For more information on the infrastructure, see [Azure Functions deployment](/learn/deployment/azure-functions/).
```

File: examples/azure-functions-deployment/bal_build.out [hunk 5]
```diff
@@ -3,10 +3,12 @@ Compiling source
         wso2/azure_functions_deployment:0.1.0
 
 Generating executable
-        @azure_functions:Function: hello, httpTriggerBlobInput, queuePopulationTimer
+        @azure_functions:Function: timer, get-hello
 
-        Execute the below command to deploy Ballerina Azure Functions:
-        az functionapp deployment source config-zip -g <resource_group> -n <function_app_name> --src <project-dir>/azure_functions_deployment/target/bin/azure-functions.zip
+        Execute the below command to deploy the function locally:
```
Comment (praneesha) [inline examples/azure-functions-deployment/bal_build.out:8]:
```suggestion
        Execute the command below to deploy the function locally.
```

File: examples/azure-functions-deployment/bal_build.out [hunk 6]
```diff
@@ -3,10 +3,12 @@ Compiling source
...
+        func start --script-root target/azure_functions --java
 
+        Execute the below command to deploy Ballerina Azure Functions:
```
Comment (praneesha) [inline examples/azure-functions-deployment/bal_build.out:11]:
```suggestion
        Execute the command below to deploy Ballerina Azure Functions.
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**